docs/deprecated-work/bri/radish34/modifying-radish34.md

```diff
@@ -4,7 +4,7 @@ The Radish34 functional proof-of-concept \(POC\) demonstrates only a few key ste
 
 But what if you wanted to add tokenized inventory, shipping & logistics, invoicing, or trade finance functions like invoice factoring?
 
-As a POC, it is certainly true that the [code](https://github.com/ethereum-oasis/baseline/tree/master/examples/radish34) of Radish34 still needs to add a layer of abstraction to take it to the next level as production-capable code for supply chain applications. This would be a good place for community members to chip in, irrespective of the additional generalization effort of going from [POC to Protocol](from-poc-to-protocol.md).
+As a POC, it is certainly true that the [code](https://github.com/ethereum-oasis-op/baseline/tree/master/examples/radish34) of Radish34 still needs to add a layer of abstraction to take it to the next level as production-capable code for supply chain applications. This would be a good place for community members to chip in, irrespective of the additional generalization effort of going from [POC to Protocol](from-poc-to-protocol.md).
```

Comment:
```suggestion
As a POC, it is certainly true that the [code](https://github.com/ethereum-oasis-op/baseline/tree/main/examples/radish34) of Radish34 still needs to add a layer of abstraction to take it to the next level as production-capable code for supply chain applications. This would be a good place for community members to chip in, irrespective of the additional generalization effort of going from [POC to Protocol](from-poc-to-protocol.md).
```